packages/example-app-base/src/components/MarkdownTable/MarkdownTable.types.ts

```diff
@@ -15,7 +15,8 @@ export interface IMarkdownTableCellProps extends IMarkdownTableProps {
    * Render the table cell as a th or td.
    * @default 'td'
    */
-  as: 'th' | 'td';
+  // TODO: doc says default is 'td', where is it set? if it has a default, why is 'as' required? making optional for now.
+  as?: 'th' | 'td';
```

Comment:
@ecraig12345 FYI. The watch issue combined with my lack of knowledge with this code made it hard to iterate and find a good fix for this.